projects/web/src/app/inicio/inicio.component.html

```diff
@@ -0,0 +1,43 @@
+<main fxLayout="column" fxLayoutAlign="start center">
+  <h1>¡Bienvenido!</h1>
```

Comment:
Maybe 'Bienvenido/a' so we include both genders in Spanish

Reply:
Is it normal / expected to add this `o/a` in primary page/document titles? I've certainly seen it before, but I'm not sure how far to go with it or how often it should be used.

Reply:
We could also use 'Bienvenidos' which even though is masculine is supposed to be considered neutral in titles.